tests/unit/sinon-sandbox-test.js

```diff
@@ -0,0 +1,61 @@
+import QUnit, { module, test } from 'qunit';
+import setupSinonSandbox, { createSandbox, restoreSandbox } from 'ember-sinon-sandbox/test-support';
+
+module('Unit | ember-sinon-sandbox');
+
+test('stores sandbox created as module property', function(assert) {
+  assert.expect(3);
+
+  assert.notOk(QUnit.config.current.testEnvironment.sandbox);
+
+  createSandbox();
```

Comment:
This test needs to call restoreSandbox (or it pollutes)

Reply:
Doh. It already bit me in terms of pollution...